marketplacetests/marketplace/pages/base.py

```diff
@@ -137,6 +129,23 @@ def show_menu(self):
             lambda m: 'mkt-nav--visible' in body.get_attribute('class'))
         return NavMenu(self.marionette)
 
+    def tap_install_button_of_first_paid_app(self):
```

Comment:
I don't find it particularly intuitive that this method results a page and an app name (we would at least need a docstring). I think I would prefer something like:

```python
def search_for_paid_apps(self):
    self.set_region('us')
    results_page = self.search(':paid')
    return results_page

def is_app_installed(self, name):
    installed = self.apps.is_app_installed(name)
    self.switch_to_marketplace_frame()
    return installed
```

```python
def test_method(self):
    acct = FxATestAccount(base_url=self.base_url).create_account()
    marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
    home_page = marketplace.launch()
    home_page.login(acct.email, acct.password)
    app = home_page.search_for_paid_apps().search_results[0]
    self.assertFalse(page.is_app_installed(app.name))
    app.tap_install_button()
    ...
```